platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -52,6 +52,15 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+<a name="java-22"></a>
+### Support for building projects with Java 22
+
+Gradle now supports [using Java 22](userguide/compatibility.html#java) for compiling, testing, and starting other Java programs.
+This can be accomplished using [toolchains](userguide/toolchains.html).
+
+Currently, you cannot run Gradle on Java 22 because Groovy lacks support for JDK 22.
```

Comment:
Even for Kotlin DSL?

Reply:
Kotlin DSL is... more complicated. Simple buildscripts might work, but projects that apply `kotlin-dsl` require manual configuration. I don't know if it's worth specifying that difference here.